packages/excel/src/row.ts

```diff
@@ -89,9 +93,44 @@ export abstract class Row {
 		return subrowOffset + SUBROW_HEADER_SIZE + baseOffset
 	}
 
-	protected unknown(opts?: ColumnSeekOptions) {
-		// NOOP - retrieve the column definition to advance to the next column position
-		this.getColumnDefinition(opts)
+	protected unknown(
+		opts?: ColumnSeekOptions,
+	): {type: ColumnDataType; value: string | number | bigint | boolean | null} {
+		const definition = this.getColumnDefinition(opts)
+		let value: string | number | bigint | boolean | null
+		switch (definition.dataType) {
```

Comment:
mmm, i'd love to avoid some of the verbosity this switch statement but i'm really not sure how best to do so without causing just-as-base duplication elsewhere.

Reply:
I agree, I'm not big fan of this verbosity because it makes it hard to maintain and it's basically a copy/paste of the individual switch/cases of each parsing method. But I didn't find a better way to do that.